modules/administration-guide/partials/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -28,24 +30,38 @@ data:
       global:
         scrape_interval:     5s             <1>
         evaluation_interval: 5s             <2>
-      scrape_configs:                       <3>  
-        - job_name: 'che'
+      scrape_configs:                       <3>
+        - job_name: 'Che Server'
           static_configs:
-            - targets: ['[che-host]:8087']  <4>          
+            - targets: ['che-host.__<{prod-short}_namespace>__:8087']  <4>
 ----
 <1> The rate at which a target is scraped.
 <2> The rate at which the recording and alerting rules are re-checked.
-<3> The Resources that Prometheus monitors. In the default configuration, a single job, `che`, scrapes the time series data exposed by {prod-short} Server.
-<4> The scrape metrics from port `8087`.
+<3> The Resources that Prometheus monitors. In the default configuration, a single job, `Che Server`, scrapes the time series data exposed by {prod-short} Server.
+<4> Scrape metrics from port `8087`. Replace `__<{prod-short}_namespace>__` with the {prod-short} namespace. The default {prod-short} namespace is `{prod-namespace}`.
 ====
 
-.Verification steps
-
-. View the metrics in the Prometheus console at `http://__<prometheus-url>__/metrics`.
+. Scale down and scale up the `Prometheus` Deployment to read the updated ConfigMap from the previous step.
++
+[subs="+attributes,quotes"]
+----
+$ {orch-cli} scale --replicas=0 deployment/prometheus -n monitoring && {orch-cli} scale --replicas=1 deployment/prometheus -n monitoring
+----
 
-. Query the metrics in the Prometheus console from `http://__<prometheus-url>__/graph`. For more information, see link:https://prometheus.io/docs/introduction/first_steps/#using-the-expression-browser[Using the expression browser].
+.Verification steps
 
-. Verify that all targets are up by viewing the targets endpoint at `http://__<prometheus-url>__/targets`.
+. Use port forwarding to access the `Prometheus` Service locally:
++
+[subs="+attributes,quotes"]
+----
+$ {orch-cli} port-forward svc/prometheus 9090:9090 -n monitoring
+----
+. Verify that all targets are up by viewing the targets endpoint at `localhost:9090/targets`.
+. Use the Prometheus console to view and query metrics:
+** View the metrics at `localhost:9090/metrics`.
+** Query metrics from `localhost:9090/graph`.
```

Comment:
If this makes sense, consider having `the` in both bullet points or in neither one.

Reply:
Updated to:
```
** The metrics are available at `localhost:9090/metrics`.
** The metrics can be queried from `localhost:9090/graph`.
```

How does this look?

Reply:
I made a last-minute suggestion. See the diff page.

You're welcome to resolve this thread.